doc/source/starting-ray.rst

```diff
@@ -0,0 +1,105 @@
+Starting Ray
+============
+
+This page covers how to start Ray on your single machine or cluster of machines.
+
+.. contents:: :local:
+
+Installation
+------------
+
+Install Ray with ``pip install -U ray``. For the latest wheels (for a snapshot of ``master``), you can use these instructions at :ref:`install-nightlies`.
+
+Starting Ray on a single machine
+--------------------------------
+
+You can start Ray by calling ``ray.init()`` in your Python script. This needs to happen before any Ray tasks or actors are called (i.e., ``function.remote()`` will not work).
+
+.. code-block:: python
+
+  import ray
+  ray.init()
+
+To stop or restart Ray, use ``ray.shutdown()``. This is useful when Ray is initialized improperly:
+
+.. code-block:: python
+
+  import ray
+  ray.init()
+  ... # ray program
+  ray.shutdown()
+
+
+To check if Ray is initialized, you can call ``ray.is_initialized()``:
+
+.. code-block:: python
+
+  import ray
+  ray.init()
+  assert ray.is_initialized() == True
+
+  ray.shutdown()
+  assert ray.is_initialized() == False
+
+See the `Configuration <configure.html>`__ documentation for the various ways to configure Ray.
+
+Using Ray on a cluster
+----------------------
+
+There are two steps needed to use Ray in a distributed setting:
+
+    1. You must first start the Ray cluster.
+    2. You need to add the ``address`` parameter to ``ray.init`` (like ``ray.init(address=...)``).
```

Comment:
```suggestion
    2. You need to add the ``address`` parameter to ``ray.init`` (like ``ray.init(address=...)``). This causes Ray to connect to the existing cluster instead of starting a new one on the local node.
```